pages/api/mint/index.ts

```diff
@@ -1,19 +1,29 @@
+/* eslint-disable class-methods-use-this */
 import axios from "axios"
+import { createHandler, Get, Query } from "next-api-decorators"
 
-export default async function handler(req, res) {
-  const { query } = req
-  const { name, description, imageUri, recipient } = query
-  try {
-    await axios.post(process.env.DEFENDER_AUTOTASK_WEBHOOK, {
-      name: name || "nft name generator",
-      description: description || "generated by ChillRx",
-      imageUri:
-        imageUri ||
-        "ipfs://bafybeidyqy7n2defa767w64g4oj4n63whgfl7mtigwqq6co3i6kg4qlo5u/lilnoun-6473.png",
-      recipient: recipient || "0xcfBf34d385EA2d5Eb947063b67eA226dcDA3DC38",
-    })
-    res.status(200).json({ name: "John Doe" })
-  } catch (err) {
-    res.status(200).json(err)
+class Mint {
+  @Get()
+  async mint(
+    @Query("name") name: string,
+    @Query("description") description: string,
+    @Query("imageUri") imageUri: string,
+    @Query("recipient") recipient: string,
+  ) {
+    try {
+      const reesponse = await axios.post(process.env.DEFENDER_AUTOTASK_WEBHOOK, {
+        name: name || "nft name generator",
+        description: description || "generated by ChillRx",
+        imageUri:
+          imageUri ||
+          "ipfs://bafybeidyqy7n2defa767w64g4oj4n63whgfl7mtigwqq6co3i6kg4qlo5u/lilnoun-6473.png",
+        recipient: recipient || "0xcfBf34d385EA2d5Eb947063b67eA226dcDA3DC38",
+      })
+      const txReceipt = JSON.parse(reesponse.data.result).hash
+      return { name: "John Doe", txReceipt }
```

Comment:
```suggestion
      return { txReceipt }
```

John can go home for the holidays 🧑‍🌾 🏡 🎄